doc/nrf/known_issues.rst

```diff
@@ -184,14 +184,14 @@ Missing :file:`CMakeLists.txt`
 Thread
 ======
 
-.. rst-class:: v1-4-99-dev1
+.. rst-class:: v1-5-0
 
 KRKNWK-8286: Faulty UART responsiveness in Thread NCP sample
   A race condition in the UARTE driver causes the retrieval of Spinel responses to fail sometimes for the :ref:`ot_ncp_sample` sample.
 
   **Workaround:** Build the sample with :option:`CONFIG_UART_0_ENHANCED_POLL_OUT` set to ``n`` or cherry-pick the upstream `Zephyr commit 2db49c`_.
 
-.. rst-class:: v1-4-99-dev1
+.. rst-class:: v1-5-0
 
 KRKNWK-8262: CoAP Client Sample crashes for nRF5340
```

Comment:
This was only affecting 1.4.99. Fixed for 1.5.

Reply:
Removed the whole issue, as v1.5.0 replaces v1.4.99-dev1.